src/event.rs

```diff
@@ -307,6 +326,18 @@ pub enum Event {
     Resize(u16, u16),
 }
 
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
+#[derive(Debug, PartialOrd, PartialEq, Eq, Clone, Copy, Hash)]
```

Comment:
Do we expect this struct to have more fields in the future? Otherwise, I would prefer flattening this and just keep the enum.

Reply:
I don't think it'll get more fields. I changed this to FocusLost and FocusGained members in the top-level Event enum. I think that's what you meant? Let me know if I misunderstood.